src/styles/stars.scss

```diff
@@ -0,0 +1,38 @@
+.catalog {
+  &__card-stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    max-width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+    &-list {
+      display: flex;
+
+      &-star {
+        width: 16px;
+        height: 16px;
+        margin-right: 4px;
+
+        @for $index from 1 through 4 {
```

Comment:
Vinicius, I see this change was due to our last chat, and I may have failed in my explanation.
The idea of using a @for would be to  make possible to change the stars count by just using a modifier at `card-stars-list` because that is what packs all stars.
The way you implemented maintains the same results than the previous iteration, always keeping 4 stars active and one inactive.

Here is my suggestion to fix the problem:
 1 - In your HTML, when you call `catalog__card-stars-list` also add a modifier, default it at 4, like so: `class="catalog__card-stars-list catalog__card-stars-list--4`, and each `star` shouldn't have a modifier. Remove "-1, -2, -3, -4, -5"
 
2 - this code here should be moved to &-list, in this way:
```
.catalog {
  &__card-stars {
    ...
    &-list {
      ...
      @for $index from 1 through 5 {
        &--#{$index} {
          .catalog__card-stars-list-star:nth-child(-n+#{$index}) {
             background: url(../images/star-active.svg) no-repeat center;
          }
        }
      }

      &-star {
        ...
        background: url(../images/star.svg) no-repeat center;
      }
    }
  }
}
```

In this way, the modifier added to the list will alter all stars to match the correct styling